StortSpelProjekt/Engine/src/Misc/Window.cpp

```diff
@@ -138,6 +141,22 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT msg, WPARAM wParam, LPARAM lParam)
 
 				SetCursorPos(500, 400);
 			}
+
+			// To click on things (temp)
+			if (DEVELOPERMODE_DEVINTERFACE == false || Input::GetInstance().GetKeyState(SCAN_CODES::ALT))
+			{
+				auto inputData = raw->data.mouse;
+				MOUSE_BUTTON button = static_cast<MOUSE_BUTTON>(inputData.usButtonFlags);
+
+				switch (button)
+				{
+				case MOUSE_BUTTON::LEFT_DOWN:
+					EventBus::GetInstance().Publish(&ButtonPressed());
+					break;
+				default:
+					break;
+				}
+			}
```

Comment:
In your user story, should you be able to click UI? Pull request name is clickable but not visible quad so I guess you want some more permanent way to click on GUI.